build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -71,9 +71,12 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
                 }
                 else
                 {
-                    // This is an old drop of latest so remove all old files to ensure a clean state
+                    Regex versionFileRegex = new Regex(@"(?<version>\d\.\d\.\d)-(?<release>.*)?");
```

Comment:
We will need that eventually though.  The Shared Framework should be publishing to its "rtm" channel while CLI will be publishing to its "preview" channel.